eng/workloads/workloads.csproj

```diff
@@ -36,15 +36,16 @@
     <PackageReference Include="Microsoft.DotNet.Build.Tasks.Installers" GeneratePathProperty="true" />
   </ItemGroup>
 
-  <!-- The list of runtimes DCP provides binaries for -->
+  <!-- The list of supported runtimes -->
```

Comment:
I find it helpful to know that this list is synchronized with a similar list in DCP.

```suggestion
  <!-- The list of supported runtimes, which matches those of DCP -->
```